examples/proc_nutation.py

```diff
@@ -27,11 +27,12 @@
 with psd.figlist_var() as fl:
     if "nScans" in s.dimlabels:
         s.mean("nScans")
+    s.rename("beta", r"$\beta$")
```

Comment:
just leave this alone -- the solution would be to make a PR in pyspecdata to `unitify_axis`